apps/aecore/lib/aecore/structures/spend_tx.ex

```diff
@@ -92,7 +93,7 @@ defmodule Aecore.Structures.SpendTx do
 
     new_sender_account_state =
       sender_account_state
-      |> deduct_fee(fee)
+      |> DataTx.deduct_fee(fee)
```

Comment:
I think this function is very similar to Account.transaction_in and out. Maybe it would be better to place it inside Aecore.Structures.Account? It operates on Account objects.